atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/DbKvs.java

```diff
@@ -1250,11 +1254,11 @@ public String getTablePrefix() {
         }
     }
 
-    private <T> T runDdl(TableReference tableRef, Function<DbDdlTable, T> runner) {
+    private void runDdl(TableReference tableRef, Function<DbDdlTable, Void> runner) {
```

Comment:
just to avoid confusion because this might not have been so clear, `Consumer<T>` imo is a better choice than `Function<T, Void>`